library/src/main/kotlin/org/quiltmc/qkl/library/brigadier/BrigadierDsl.kt

```diff
@@ -57,11 +58,13 @@ public fun <S> CommandDispatcher<S>.register(
  *
  * @see CommandBuildContext
  */
+@Suppress("MISSING_DEPENDENCY_CLASS")
```

Comment:
Note: don't release this until re-built against a mappings release QuiltMC/quilt-mappings#307